about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>about</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="about.css">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+      integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V"
+      crossorigin="anonymous"
+    />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+<body>
+    <section class="about-section">
+        <div class="headline-img1"></div>
+        <div class="secondary-cta">
+            <ul class="font-face second-cta container">
+                <li class="s-list"><a class="social" href="#"><i class="fa fa-facebook social" aria-hidden="false"></i></a></li>
+                <li class="s-list"><a class="social" href="#"><i class="fa fa-twitter"></i></a></li>
+                <li class="s-list"><a href="#">English</a></li>
+                <li class="s-list"><a href="#">My Page</a></li>
+                <li class="s-list"><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <header>
+            <nav class="container">
+                <div class="logoDiv">
+                    <a class="logo font-face" href="index.html">&lt;/&gt;CodeWorld</a>
+                </div>
+
+                <img class="mobile-menu pb-4" id="mobileMenu" src="images/ic_menu.svg" alt="Open Navigation">
+                
+                <div class="nav-list">
+                    <span id="mobile-exit">&times;</span>
+                    <ul>
+                        <li class="p-cta active"><a href="about.html">About</a></li>
+                        <li class="p-cta"><a href="#">Program</a></li>
+                        <li class="p-cta"><a href="#">join</a></li>
+                        <li class="p-cta"><a href="#">Sponsor</a></li>
+                        <li class="p-cta"><a href="#">News</a></li>
+                        <li class="p-cta call2action"><a href="#">Start Now</a></li>
+                    </ul>
+                </div>
+            </nav>
```

Comment:
- [ ] [REQUIRED] For the `navbar`, kindly justify the contents, ensured that the contents are properly aligned just as it is on the [behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

### Your Implementation
![Screenshot from 2022-04-21 17-12-25](https://user-images.githubusercontent.com/58692590/164506158-29861291-cd1a-4862-adf9-0d09f9ca4293.png)

### Behance Design
![Screenshot from 2022-04-21 17-12-14](https://user-images.githubusercontent.com/58692590/164506132-dfda1e17-2228-4a86-9602-ac9eb3c886e8.png)